### PR TITLE
Add a log message when deleting test cases in grouper.

### PR DESCRIPTION
**Pull request by mbarbella-chromium**
Trivial change, PTAL when you have time. This is the only obvious case I see where we aren't logging after deleting a test case, but there may be bulk cases I missed.

### CODE REVIEW

File: src/appengine/handlers/cron/grouper.py [hunk 1]
```diff
@@ -99,6 +99,7 @@ def group_testcases():
     # Remove duplicates early on to avoid large groups.
     if (not testcase.bug_information and not testcase.uploader_email and
         has_testcase_with_same_params(testcase, testcase_map)):
+      logs.log('Deleting duplicate testcase %s.' % testcase_id)
```
Comment (Dor1s) [inline src/appengine/handlers/cron/grouper.py]: Is `testcase_id` a string, not an integer?

Comment (mbarbella-chromium) [reply src/appengine/handlers/cron/grouper.py]: Looks like get_open_testcase_id_iterator yields integers. I'll switch this to %d.

**Pull request merged**